app/controllers/admin/routes.py

```diff
@@ -1,16 +1,50 @@
-from flask import request, render_template
-from flask import Flask, redirect, flash
-from app.controllers.admin.createEvents import createEvent
+from flask import request, render_template, abort, flash
+from flask import Flask, redirect, url_for, g
+from app.models.event import Event
+from app.models.programEvent import ProgramEvent
+from app.controllers.admin import admin_bp
+from app.models.eventParticipant import EventParticipant
+from app.models.programEvent import ProgramEvent
 from app.models.program import Program
 from app.models.term import Term
-from app.controllers.admin import admin_bp
 from app.logic.getAllFacilitators import getAllFacilitators
-from flask import g
+from app.controllers.admin.createEvents import createEvent
+from app.logic.updateTrackVolunteers import getEventLengthInHours
+from app.controllers.admin.changeTrackVolunteer import searchTrackVolunteers
+
 
 @admin_bp.route('/testing_things', methods=['GET'])
 def testing():
     return "<h1>Hello</h1>"
 
+@admin_bp.route('/<programID>/<eventID>/track_volunteers', methods=['GET'])
+def trackVolunteersPage(programID, eventID):
+    if g.current_user.isCeltsAdmin:
+        if ProgramEvent.get_or_none(ProgramEvent.event == eventID, ProgramEvent.program == programID):
+            eventParticipantsData = EventParticipant.select().where(EventParticipant.event == eventID)
+
+            eventParticipantsData = eventParticipantsData.objects()
+
+            event = Event.get_by_id(eventID)
+            program = Program.get_by_id(programID)
+            startTime = event.timeStart
+            endTime = event.timeEnd
+            eventDate = event.startDate #start date and end date will be the same
```

Comment:
I don't think there's a good reason for 3 separate variables here. They aren't used anywhere else other than the next line, and the original sources aren't that long.